go/vt/vttablet/tabletserver/planbuilder/dml.go

```diff
@@ -117,9 +116,16 @@ func analyzeDelete(del *sqlparser.Delete, tables map[string]*schema.Table) (plan
 		plan.Reason = ReasonTable
 		return plan, nil
 	}
-	table, err := plan.setTable(tableName, tables)
-	if err != nil {
-		return nil, err
+	table, tableErr := plan.setTable(tableName, tables)
```

Comment:
nit - why not just keep `err`, instead of `tableErr`?

Reply:
I just wanted it to be clear _which_ error the conditional behavior is based on so giving it a more specific name made more sense.